examples/framework-lit/astro.config.mjs

```diff
@@ -1,7 +1,10 @@
 import { defineConfig } from 'astro/config';
 
-// https://astro.build/config
+import litPlugin from '@astrojs/lit';
+import tailwindPlugin from '@astrojs/tailwind';
+import sitemapPlugin from '@astrojs/sitemap';
+import partytownPlugin from '@astrojs/partytown';
+
 export default defineConfig({
-	// Enable the lit renderer to support LitHTML components and templates.
-	renderers: ['@astrojs/renderer-lit'],
+	integrations: [litPlugin(), tailwindPlugin(), sitemapPlugin(), partytownPlugin()],
```

Comment:
Just curious why all of these integrations are added to the framework-lit example? Should these maybe be in framework-multiple instead? The other `framework-` examples don't have these non-framework integrations.

Reply:
Yes! This is just a playground for now for anyone who checks out this PR. I'll revert this soon (do we want a "with-integrations" example?)